src/localConstants/sdkDappCoreConstants.ts

```diff
@@ -0,0 +1,10 @@
+export {
+  TRANSACTIONS_ENDPOINT,
```

Comment:
it's correct to get them from the package, but maybe export them from lib/sdkDappCore ? 
the name localConstants seems to refer to local constants defined by the user, not re-exported (in my oppinion)